integration-test/integration-utils.ts

```diff
@@ -6,6 +6,12 @@ export const LF_HOST = process.env.LF_HOST ?? "http://localhost:3000";
 export const LF_PUBLIC_KEY = process.env.LF_PUBLIC_KEY ?? "pk-lf-1234567890";
 export const LF_SECRET_KEY = process.env.LF_SECRET_KEY ?? "sk-lf-1234567890";
 
+export const getKeys = {
```

Comment:
While refactoring this, should we rename getKeys and getHeaders to make clear that they are not functions?
Or actually declare them as functions and invoke them?
At the moment the naming is a bit confusing.

Reply:
I think renaming them would be a better options. Maybe something like `langfuseKeys`. WDYT @ksjogo ?

Reply:
I am usually in favor of actually declaring things as functions as that's easier to refactor in the future if needs change.

Reply:
True! Will do that!

Reply:
Updated! Kindly review @ksjogo